### PR TITLE
6437 add buyer ordering claim

### PR DESCRIPTION
**Pull request by SimonLLawrence**
Added Ordering Manage claim for Buyer

### CODE REVIEW

File: src/NHSD.BuyingCatalogue.Identity.Api/Services/ProfileService.cs [hunk 1]
```diff
@@ -30,7 +30,8 @@ public sealed class ProfileService : IProfileService
                     }
                 },
                 {
-                    OrganisationFunction.Buyer,  new List<Claim>
+                    OrganisationFunction.Buyer, 
+                    new List<Claim>
```
Comment (sam-roberts-bjss) [inline src/NHSD.BuyingCatalogue.Identity.Api/Services/ProfileService.cs:34]: Sorry Simon I wasn't clear in regards to my spacing comment. I was referring to the line `new Claim(ApplicationClaimTypes.Ordering,ApplicationPermissions.Manage)`. With no space after the `.Ordering,` I find it difficult to read. Again this comes down to needing a review of our tooling e.g. Resharper settings

Comment (SimonLLawrence) [reply src/NHSD.BuyingCatalogue.Identity.Api/Services/ProfileService.cs:34]: Ok, Just fixing them now , not claims are in the list so adding the missing claims

Comment (SimonLLawrence) [reply src/NHSD.BuyingCatalogue.Identity.Api/Services/ProfileService.cs:34]: Ok will fix that , i  have a fix for the integration tests too.

**Comment by SimonLLawrence** [comment]
I so used to auto format on save sorting formatting issues out.

**Pull request merged**